.circleci/config.yml

```diff
@@ -127,6 +127,7 @@ commands:
             - run:
                 environment:
                   DD_TRACE_AGENT_URL: http://localhost:9126
+                  DD_TESTING_RAISE: true
```

Comment:
do we need `--pass-env` on `riot` for the `unless` condition ?

Reply:
ah nice catch! I think we'd need it; is there a reason why it's not currently used there?

Reply:
not that I can think of

Reply:
https://github.com/DataDog/dd-trace-py/pull/2500